src/components/Actions.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function Actions(props) {
```

Comment:
Here, you don't need to call props since we are not sending information to this component